src/Microsoft.ML.OnnxTransformer/OnnxTransform.cs

```diff
@@ -37,7 +37,7 @@ namespace Microsoft.ML.Transforms.Onnx
     /// <summary>
     /// <see cref="ITransformer"/> resulting from fitting an <see cref="OnnxScoringEstimator"/>.
     /// </summary>
-    public sealed class OnnxTransformer : RowToRowTransformerBase
+    public sealed class OnnxTransformer : RowToRowTransformerBase // MYTODO: Should I consider not to inherit from this, since now OnnxTransformer would be able to drop columns and not use the RowToRowMapperTransform?
```

Comment:
The `RowToRowTransformerBase` clearly states that it's only supposed to add columns, not drop or modify any input columns. The whole design of that class, and its use of RowToRowMapperTransform, enforces this. Having to hack it by changing some of its methods into virtual methods feels somewhat hacky and incorrect.

So I would appreciate other thoughts regarding if it's okay to leave this as it is?

I guess I could simply not inherit from that class, and copy its code in here. But, is it worth it? Also, I might be concerned about backward compatibility with ML.NET models that were saved to disk and included OnnxTransformers; I would still need to test if removing this inheritance would affect loading back those models.